.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - uses: julia-actions/julia-processcoverage@v1
       - uses: codecov/codecov-action@v1
```

Comment:
It looks like this is still using v1, so `file` should still work in this case. Perhaps we should update this to v2 since v1 is no longer supported upstream?

Reply:
Thanks for  the suggestion, I also updated the julia version to 1.8.2. Are you agree on that ?

Reply:
The Julia version doesn't need to be updated, actually; the `1` refers to the latest stable release, so `1` is the same as `1.8.2`. Similarly, `1.0` refers to the latest `1.0.x`. While 1.0 is no longer supported by the Julia project, the minimum Julia version supported by this package is 1.0 (see [here](https://github.com/JuliaLang/Example.jl/blob/928ba3c291f9a3ba227bd01735039fbfcf941b5c/Project.toml#L6) but note that `1` in Project.toml is different from `1` in GitHub Actions) and it's best to test a package with the earliest Julia version it supports.

All that is to say that I think you can limit the changes in this PR to updating the Codecov Action version to `v2` and replacing `file` with `files` since the rest is already working as expected. 🙂

Reply:
Okay thanks for clear that out. The problem is that with the `1` Julia version I obtain this error:
```bash
 Error: Unable to resolve action `julia-actions/julia-processcoverage@v2`, unable to find version `v2`
 ```

Reply:
Sorry, I was unclear! The `v2` is for the Codecov action, not the coverage processing action. That is, `uses: codecov/codecov-action@v1` should now be `uses: codecov/codecov-action@v2`.

Reply:
Done ☂️